arviz/plots/tsplot.py

```diff
@@ -0,0 +1,430 @@
+"""Plot timeseries data."""
+import warnings
+import numpy as np
+
+from ..sel_utils import xarray_var_iter
+from ..rcparams import rcParams
+from .plot_utils import default_grid, get_plotting_function
+
+
+def plot_ts(
+    idata,
+    y,
+    x=None,
+    y_hat=None,
+    y_holdout=None,
+    y_forecasts=None,
+    x_holdout=None,
+    plot_dim=None,
+    holdout_dim=None,
+    num_samples=100,
+    backend=None,
+    backend_kwargs=None,
+    y_kwargs=None,
+    y_hat_plot_kwargs=None,
+    y_mean_plot_kwargs=None,
+    vline_kwargs=None,
+    textsize=None,
+    figsize=None,
+    legend=True,
+    grid=False,
```

Comment:
same comment I did for `plot_lm`. Is the grid *vital* to the plot? The default should be to use or not use a grid based on the matplotlib style users have chosen, not only it's easy to set, it will affect all plots at once without needing any extra argument nor call to `plt.` after calling ArviZ. Adding grid here means we want the grid to be/not be present and believe this to be important enough to overwrite the defaults set by the user.

Reply:
Hmm, Didn't think it in "matplotlib style" way! I do not feel the need of it now.